src/utils/sources-tree.js

```diff
@@ -73,14 +73,31 @@ function createParentMap(tree: any): WeakMap<any, any> {
  * @memberof utils/sources-tree
  * @static
  */
+function getFilenameFromPath(pathname) {
+  let filename = "";
+  if (pathname) {
```

Comment:
And here, with flow checking that `pathname` is always a string you don't really need this check unless you're intentionally looking for the empty string.